homeassistant/components/zha/strings.json

```diff
@@ -730,6 +730,12 @@
       },
       "presence_detection_timeout": {
         "name": "Presence detection timeout"
+      },
+      "max_heat_setpoint_limit": {
+        "name": "Max Heat Setpoint Limit"
```

Comment:
I think this should rather be "Max heat setpoint limit", and the same everywhere for the translations

Reply:
Done